src/components/appheadernavigation.main.jsx

```diff
@@ -101,72 +100,48 @@ class AppHeaderNavigationMain extends React.Component {
     isOfflineCheck(isOfflineValue);
   }
 
-  renderCategories() {
+  render() {
     const { navigations } = this.state;
-    return navigations.map((category) => {
-      if (category._child) {
-        return (
-          <li className="nav-item dropdown" key={category.name} data-name={category['display-name']} data-el-container="category-nav-item-container">
-            <a className="nav-link dropdown-toggle" href="/" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-              {category['display-name']}
-            </a>
-            <div className="dropdown-menu sub-category-dropdown-menu" aria-label="navbarDropdown">
-              {category._child.map(subcategory => (
-                <Link to={`/category/${encodeURIComponent(subcategory.self.uri)}`} key={subcategory.name} className="dropdown-item" id={`header_navbar_sub_category_button_${subcategory.name}`} data-target=".navbar-collapse" title={subcategory['display-name']}>
-                  <div data-toggle="collapse" data-target=".navbar-collapse">
-                    <span>
-                      {subcategory['display-name']}
-                    </span>
-                  </div>
-                </Link>
-              ))}
-            </div>
-          </li>
-        );
-      }
-      return (
-        <li key={category.name} data-name={category['display-name']} data-el-container="category-nav-item-container">
-          <Link to={`/category/${encodeURIComponent(category.self.uri)}`} className="nav-item" id={`header_navbar_category_button_${category.name}`} title={category['display-name']}>
-            <div data-toggle="collapse" data-target=".navbar-collapse">
-              <span>
-                {category['display-name']}
-              </span>
-            </div>
-          </Link>
-        </li>
-      );
-    });
-  }
+    const { isMobileView } = this.props;
 
-  render() {
     return (
-      <div className="main-nav-container" id="header_navbar_container" data-region="mainNavRegion" style={{ display: 'block' }}>
```

Comment:
this id is used in e2e tests

Reply:
I've changed selectors in the test, so that's not an issue anymore.